Library/Formula/flyway.rb

```diff
@@ -0,0 +1,16 @@
+class Flyway < Formula
+  homepage "http://flywaydb.org/"
+  url "https://bintray.com/artifact/download/business/maven/flyway-commandline-3.2.1-macosx-x64.tar.gz"
```

Comment:
Can we use the version without the embedded ` JRE `? That way we're not limited to only being able to install this on ` x64 ` platforms, and we're not duplicating people's Java installs... Which is useful, given the size of Java.

Adding a ` depends_on :java ` to the formula will ensure users have Java available to the tool.